rpcs3/Gui/GameViewer.cpp

```diff
@@ -233,8 +233,10 @@ void GameViewer::RemoveGame(wxCommandEvent& event)
 {
 	long i = GetFirstSelected();
 	if (i < 0) return;
-	
-	fs::remove_all(vfs::get(m_path) + this->GetItemText(i, 6).ToStdString());
+
+	wxMessageDialog *dial = new wxMessageDialog(NULL, wxT("Permanently delete game files?"), wxT("Confirm Delete"), wxYES_NO | wxNO_DEFAULT);
```

Comment:
I recently started using wxWidgets and from what I understand, whenever an object is closed (ie. a message dialog being closed), then even if you used new to create it, it automatically gets freed by wxWidgets.